Assets/Scripts/GameEnums.cs

```diff
@@ -9,9 +9,14 @@ public enum CommandTypes
         DEFEND,
         PATROL,
     }
-    public enum EnemyTypes
+    public enum EnemyTarget
     {
         PLAYER,
         UNIT,
     }
+    public enum EntityType
```

Comment:
Not sure if we even need to differentiate between contact and distance damage considering that can be determined with the attack range of a unit, but should be fine to keep this.

That said, I'd suggest a rename, to something like `EntityRangeType`